frontend/src/components/TransferCreditHolder.tsx

```diff
@@ -68,6 +76,58 @@ const Wrapper = styled.div`
   height: 100%;
 `;
 
+/**
+ * Determines the corresponding transferable exam type and name for the given NEU course.
+ * @param examGroups The transferable exam groupings (either AP or IB).
+ * @param course The NEU course to query against.
+ * @returns The exam type and name as a string.
+ */
+function findExamText(
+  examGroups: TransferableExamGroup[],
+  course: ScheduleCourse
+) {
+  for (const examGroup of examGroups) {
+    for (const transferableExam of examGroup.transferableExams) {
+      for (const mappableCourse of transferableExam.mappableCourses) {
+        if (
+          mappableCourse.subject === course.subject &&
+          String(mappableCourse.classId) === course.classId
+        ) {
+          return transferableExam.type + " " + transferableExam.name;
```

Comment:
kinda nit but let's try using string interpolation instead of "+" xDDD
so in this case it would be return `${transferableExam.type} ${transferableExam.name}`